Non-Interactive-Protocol.md

```diff
@@ -11,115 +11,111 @@
-1. The funding transaction doesn't confirm, a party must either bump its feerate
-to confirm before DLC maturing or double-spend its funding collateral, thus cancelling the
-DLC.
+1. The funding transaction doesn't confirm, and so a party must either bump its fee-rate
+to confirm before DLC maturation or else double-spend its funding collateral canceling the DLC.
 
-2.  The funding transaction confirms, the oracle releases the outcome signature, a
-participant must broadcast the DLC transaction paired with the outcome.
+2.  The funding transaction confirms, and the oracle releases the outcome signature, so a
+participant must broadcast the CET corresponding to the oracle outcome.
 
-3. The funding transaction confirms, oracle doesn't release the outcome signature, independently
-of counterparty behavior, the party must broadcast the refund transaction after its timelock
-expiration.
+3. The funding transaction confirms, and the oracle doesn't release an outcome signature,
+so a party must broadcast the refund transaction after its time-lock expiration.
 
 # General Requirements
 
 A DLC client should have censorship-resistant access to the blockchain, use a
-local fee-estimator and ensure it's well-connected to the tx-relay network.
+local fee-estimator and ensure it is well-connected to the tx-relay network.
 
-We recommend to not react on any mempool event as otherwise it would be a source of
+We recommend clients should register no callbacks on mempool events as doing so may be a source of
 cross-layer mapping. <sup>[Cross-layer Mapping](https://arxiv.org/pdf/2007.00764.pdf)</sup>
 
-We recommend to not produce local non-adaptor signatures for transaction until their broadcast
-is required to avoid their misuage in case of site compromise, assuming an external signer.
+We recommend clients should not produce local (non-adaptor) signatures for CETs until they are about to be broadcast
+in order to avoid their misuse in case of site compromise, assuming an external signer.
 
-Further, we recommend duplicate backups of counterparty signatures to avoid relying on
-counterparty honest behavior in case of primary storage failure.
+Furthermore, we recommend duplicated backups of counter-party signatures be stored to avoid relying on
+honest counter-party behavior in the case of primary storage failure.
 
-Transaction finalization is not implying a mandatory key material access.[ FIXME: Client Key Management. ]
+Transaction finalization does not imply a mandatory key material access. [ FIXME: Client Key Management. ]
 
 # Fee bumping policy
 
-An onchain state machine must take actions based on different timers parameterized by three
-values: `congestion_bump_frequency`/`security_bump_frequency`/`security_point`/`refund_security_point`.
-Their definition and setting recommendations are laid out in this section, their usage is precised
-in following sections.
+An on-chain state machine must take actions based on different clocks parameterized by three
```

Comment:
Suggestion: "An state machine taking as input on-chain events" (on-chain state machine sounds weird to me)